doc/BUILD.md

```diff
@@ -4,7 +4,7 @@ How to build boot2docker.iso locally
 The boot2docker.iso is built with Docker, via a Dockerfile.
 
 During `docker build` we
-* fetch, patch with AUFS support and build the 3.15.3 Linux Kernel with Tiny Core base configuration
+* fetch, patch with AUFS support and build the 4.4.100 Linux Kernel with Tiny Core base configuration
```

Comment:
Ouch, nice catch!  Since we update the kernel relatively frequently and I usually stay on top of updating `README.md`, I think I'd rather adjust this to drop the version number, and interested folks can dig into either the `Dockerfile` or the `README` to find out the specific version number.